website/common/script/constants.js

```diff
@@ -28,7 +28,7 @@ export const SUPPORTED_SOCIAL_NETWORKS = [
 
 export const GUILDS_PER_PAGE = 30; // number of guilds to return per page when using pagination
 
-export const PARTY_LIMIT_MEMBERS = 30;
+export const PARTY_LIMIT_MEMBERS = 29;
```

Comment:
This is causing an integration test to fail--I understand from the PR description what this is for, but you'll need to update the expectation in the test suite!